backend/app/api/endpoints/knowledge.py

```
@@ -767,3 +810,173 @@ def get_qa_history(
+    db: Session = Depends(get_db),
+):
+    """
+    Manually refresh knowledge base summary.
+
+    Triggers regeneration of the knowledge base summary based on
+    aggregated document summaries. Runs in background.
+    """
+    from app.schemas.summary import SummaryRefreshResponse
+    from app.services.summary_service import get_summary_service
+
+    summary_service = get_summary_service(db)
```

Ruff (0.14.10)

Local variable `summary_service` is assigned to but never used

Remove assignment to unused variable `summary_service`

(F841)